### PR TITLE
Fix/recurring detail container

### PR DESCRIPTION
**Pull request by wboereboom**
**Description**
RecurringDetails container class is not yet contained in the API specs. Had to edit models manually to include this container class

**Fixed issue**:  #385

### CODE REVIEW

File: src/__tests__/recurring.spec.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ afterEach(() => {
 });
 
 describe("Recurring", (): void => {
-    test.each([false, true])("should test have recurring details list, isMock: %p", async (isMock): Promise<void> => {
+    test.each([true])("should test have recurring details list, isMock: %p", async (isMock): Promise<void> => {
```
Comment (michaelpaul) [inline src/__tests__/recurring.spec.ts:69]: Here it's a shorter version for this test:
```suggestion
    test("should list recurring details ", async (): Promise<void> => {
        scope.post("/listRecurringDetails")
            .reply(200, listRecurringDetailsSuccess);
        const request = createRecurringDetailsRequest();
        
        const result = await recurring.listRecurringDetails(request);
        
        expect(result).toBeTruthy();
        expect(result.details?.[0].RecurringDetail.recurringDetailReference).toBe('recurringReference');
    });
```

File: src/typings/recurring/recurringDetailContainer.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * The version of the OpenAPI document: 49
+ * Contact: developer-experience@adyen.com
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
```
Comment (michaelpaul) [inline src/typings/recurring/recurringDetailContainer.ts:8]: In this case, this file was not auto generated. Should we state that here?

File: src/typings/recurring/recurringDetailContainer.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import {RecurringDetail} from "./recurringDetail";
+
+export class RecurringDetailContainer {
+    'RecurringDetail'?: RecurringDetail;
```
Comment (michaelpaul) [inline src/typings/recurring/recurringDetailContainer.ts:16]: Can we make this field required? 
```suggestion
    'RecurringDetail': RecurringDetail;
```

**Review by michaelpaul — CHANGES_REQUESTED** [review]

File: src/typings/recurring/recurringDetailContainer.ts (hunk 2 above)
Comment (wboereboom) [reply src/typings/recurring/recurringDetailContainer.ts:8]: That is a good question. We do not have clear guidelines on that yet.

**Review by michaelpaul — APPROVED** [review]

**Review by zaiddreakh — APPROVED** [review]

**Pull request merged**